src/components/sectionList/listActions/DefaultListActions.tsx

```diff
@@ -9,12 +9,14 @@ type DefaultListActionProps = {
     model: ModelWithAccess
     onShowDetailsClick: (model: ModelWithAccess) => void
     onOpenSharingClick: (id: string) => void
+    onDeleteClick: () => void
 }
 
 export const DefaultListActions = ({
     model,
     onShowDetailsClick,
     onOpenSharingClick,
+    onDeleteClick,
 }: DefaultListActionProps) => {
     const editAccess = canEditModel(model)
```

Comment:
Probably needs a `canDeleteModel` here as well.

I've opted to put this in functions and not use the direct `access.write` property here, because this stuff could change in the future. Mostly for `edit` - because there's `update: false, write: false` on the object, so I didn't want to that directly in the component.

Reply:
Done, see above